Key Path

Key Path isn't working? Although I'm not 100% sure how to set it up, it states something is missing. I tried npm keypath install, but wasn't sure what to do next.

Okay. so I did npm start -- [--query=botw] [--interval=minutes: 10] and it's giving me the same error as before: Error: Cannot find module 'es6-polyfills/lib/polyfills/object'

> twitter-picbot@1.1.1 start C:\Users\hidas\Downloads\twitter-picbot-develop\twitter-picbot-develop
> node app/index.js "--query=botw" "--interval='0" "*/10" "*" "*" "*" "*'"

module.js:491
    throw err;
    ^

Error: Cannot find module 'es6-polyfills/lib/polyfills/object'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at C:\Users\hidas\Downloads\twitter-picbot-develop\twitter-picbot-develop\node_modules\loglevel-message-prefix\lib\main.js:40:34
    at Object.<anonymous> (C:\Users\hidas\Downloads\twitter-picbot-develop\twitter-picbot-develop\node_modules\loglevel-message-prefix\lib\main.js:45:2)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! twitter-picbot@1.1.1 start: `node app/index.js "--query=botw" "--interval='0" "*/10" "*" "*" "*" "*'"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the twitter-picbot@1.1.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\hidas\AppData\Roaming\npm-cache\_logs\2017-10-05T05_48_38_866Z-debug.log

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 1.1.2
+
+- Fixed missing es6-polyfill for outdated lib loglevel-message-prefix
+
 ## 1.1.1
 
 - Fixed url and src

--- a/package.json
+++ b/package.json
@@ -1,8 +1,9 @@
 {
   "name": "twitter-picbot",
-  "version": "1.1.1",
+  "version": "1.1.2",
   "dependencies": {
     "client-oauth2": "1.0.0",
+    "es6-polyfills": "2.0.0",
     "loglevel": "1.4.0",
     "loglevel-message-prefix": "1.0.0",
     "node-cron": "1.0.0",

